src/goto-programs/abstract-interpretation/wrapped_interval.h

```diff
@@ -1425,6 +1427,9 @@ class wrapped_interval : public interval_templatet<BigInt>
 private:
   static BigInt compute_upper_bound(const type2tc &t)
   {
+    if (is_pointer_type(t))
+      return BigInt::power2(32);
```

Comment:
The constant 32 here looks fishy. Consider replacing it with `pointer_width()` or `address_width` from config.ansi_c - depending on the meaning of "upper bound" here: what's the order relation considered for wrapped intervals on pointers?

Reply:
> The constant 32 here looks fishy. 

You are right, 32 should be changed.

> what's the order relation considered for wrapped intervals on pointers

At this point it would not be the pointer, just its address value (guess this answer it) so `if(ptr != NULL)` or `((size_t)ptr == 0x42)` could be eventually processed. 

We are still not tracking any pointers though, the change was needed because the wrapped domain always needs an "upper bound".